src/components/ui/Panel.jsx

```diff
@@ -237,7 +237,7 @@ export default class Panel extends React.Component {
               onClick={() => this.handleHeaderClick()}
               {...(isMobile && forceResizeHandlers)}
             >
-              {resizable && size === 'minimized' && minimizedTitle
+              {resizable && isMobile && size === 'minimized' && minimizedTitle
```

Comment:
Maybe this `isMobile` check should be defined in the `onClick` action ?  
It does not make sense to update the `size` state on desktop, right ?

Reply:
:+1: